ingestion_server/ingestion_server/cleanup.py

```diff
@@ -55,6 +55,9 @@
 # Filter out low-confidence tags, which indicate that the machine-generated tag
 # may be inaccurate.
 TAG_MIN_CONFIDENCE = 0.90
+# Filter out tags that match the following providers (either because they haven't
+# been vetted or because they are known to be low-quality).
+TAG_SKIP_PROVIDERS = {"rekognition"}
```

Comment:
The name is kind of confusing to me. Could we change it to something like 
`EXCLUDED_TAG_PROVIDERS` or `FILTERED_TAG_PROVIDERS` ?

Reply:
I had the hardest time naming this 😆 I wanted `TAG` at the front to match the previous constant, but this fits much better!